### PR TITLE
Support Jade syntax highliting

### PR DESCRIPTION
**Pull request by KindDragon**

### CODE REVIEW

File: src/JSON/Schema/suggestions.json [hunk 1]
```diff
@@ -121,6 +121,12 @@
       "link": "https://visualstudiogallery.msdn.microsoft.com/4cd59e4a-82e8-4b4e-8302-d102fc81b090",
       "fileTypes": [ "karma.conf.js", ".spec.ts" ]
     },
+    "Node.js Tools for Visual Studio 2015": {
+      "description": "Syntax highlightning for Jade",
+      "productId": "??? don't know it's msi extension",
```
Comment (KindDragon) [inline src/JSON/Schema/suggestions.json:126]: What I should write there?

Comment (KindDragon) [inline src/JSON/Schema/suggestions.json:126]: c:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\Extensions\Microsoft\Node.js Tools for Visual Studio\1.1\ 
[extension.vsixmanifest](https://github.com/madskristensen/SolutionExtensions/files/219077/extension.vsixmanifest.txt)

**Comment by madskristensen** [comment]
Unfortuantely .MSIs are not supported by this extension. Visual Studio doesn't allow silent installs of MSI packages. Only .vsix files are supported

**Comment by KindDragon** [comment]
Can you install msi not silently? :-)
Maybe ask user before that

**Comment by madskristensen** [comment]
I can't because most MSI's needs elevated permissions and so it wouldn't work if you don't run VS as admin

**Comment by KindDragon** [comment]
Ok, but it's sad :-)

**Pull request closed without merging**